lib/gui/os/open-external/open-external.js

```diff
@@ -21,9 +21,23 @@
  */
 
 const angular = require('angular');
+const url = require('url');
+
 const MODULE_NAME = 'Etcher.OS.OpenExternal';
 const OSOpenExternal = angular.module(MODULE_NAME, []);
 OSOpenExternal.service('OSOpenExternalService', require('./services/open-external'));
 OSOpenExternal.directive('osOpenExternal', require('./directives/open-external'));
 
+OSOpenExternal.run(function(OSOpenExternalService) {
+  document.addEventListener('click', (event) => {
+    const target = event.target;
+    if (target.tagName === 'A' && angular.isDefined(target.href)) {
```

Comment:
@jviotti regarding https://github.com/resin-io/etcher/pull/1043#discussion_r98483565

Didn't actually knew this, but when `href` is `#`, `target.href` is `file:///Users/stefanmihaila/resin.io/etcher/lib/gui/index.html#` on my machine. For `aaa`, it is `file:///Users/stefanmihaila/resin.io/etcher/lib/gui/aaa`.

I decided to use `OSOpenExternalService.open` just for `http` and `https` external links.

Do you think it's worth adding tests for this? If so, what's a good way to do it? Expose the event handler on `OSOpenExternal` and test it?